agent/functional_tests/util/utils_unix.go

```diff
@@ -133,6 +134,12 @@ func RunAgent(t *testing.T, options *AgentOptions) *TestAgent {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	if options != nil && options.EnableTaskENI {
+		// if task networking is enabled, needs to wait for container instance to become active
+		err = agent.WaitContainerInstanceStatus("ACTIVE")
```

Comment:
Can this be skipped if the instance type does not support trunking?

Reply:
we can do that. but i'm trying to avoid more instance identity document dependency as per [this comment](https://github.com/aws/amazon-ecs-agent/pull/1961#discussion_r279854839). i think it's probably fine to add two more DescribeContainerInstances call for a few tests (currently around 5?) to avoid that dependency? the test can pass either way..

Reply:
Umm, your call, totally.